install.ts

```diff
@@ -57,20 +54,20 @@ function pinup(...args: any): void {
 }
 
 async function follow(url: string): Promise<any> {
-  let located: boolean = false;
   let res: any; // TODO: annotate deno Response
+  let located: boolean = false;
   let count: number = 0;
   while (!located) {
-    if (++count === 4) {
-      panic(Error(`Unable to fetch from ${url}`));
-    }
     res = await fetch(url);
     if (res.status >= 300 && res.status < 400) {
+      if (++count === 4) {
```

Comment:
>= may be a safer choice here.

Should 4 be named? E.g. MAX_FOLLOW_DEPTH, MAX_FOLLOWS or something?

Reply:
`>=`

Reply:
yea, the 4 is too magic